eventio/simtel/objects.py

```diff
@@ -1,6 +1,6 @@
 ''' Implementations of the simtel_array EventIO object types '''
 import numpy as np
-from io import BytesIO
+from io import BytesIO, SEEK_END
```

Comment:
I'm on the phone, and somehow do not see where SEEK_END is needed....

Reply:
removed it